src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Extensions/SyntaxNodeExtensions.cs

```diff
@@ -289,6 +289,7 @@ public static SyntaxList<AttributeListSyntax> GetAttributeLists(this SyntaxNode?
                 AccessorDeclarationSyntax accessor => accessor.AttributeLists,
                 ParameterSyntax parameter => parameter.AttributeLists,
                 CompilationUnitSyntax compilationUnit => compilationUnit.AttributeLists,
+                TypeParameterSyntax typeParameter => typeParameter.AttributeLists,
```

Comment:
This is missing StatementSyntax as well.

Reply:
Great catch!